drivers/hmis/lib/hmis_data_cleanup/util.rb

```diff
@@ -17,8 +17,18 @@ class Util
     # Remove all ExportIDs from HMIS Enrollments.
     # This should be done after an HMIS migration, otherwise ServiceHistoryService generation will behave incorrectly
     def self.clear_enrollment_export_ids!
-      without_papertrail_or_timestamps do
-        Hmis::Hud::Enrollment.hmis.update_all(ExportID: nil)
+      Hmis::Hud::Enrollment.hmis.where.not(ExportID: nil).find_in_batches(batch_size: 5_000) do |batch|
+        batch.each { |enrollment| enrollment.ExportID = nil }
+
+        result = Hmis::Hud::Enrollment.import(
```

Comment:
Maybe I should switch this back to update_all or update_columns for perf reasons? this one will be large as its nearly all enrollments.

I'm not sure what is the most performant way to do that. I thought it was import. @ttoomey thoughts on this? In this case it is nullifying one column.

Reply:
This should be fine, `import` with a conflict key (maybe that's missing, or maybe it defaults to id?) will do an upsert/update_all.

Reply:
oh, the `conflict_target` is there, I just missed it.

Reply:
I'd suggest reverting this change. Using `Hmis::Hud::Enrollment.hmis.update_all(ExportID: nil)` is just one sql update statement. It doesn't get much more performant than that.

I think the only reason to use a technique like this "upsert on PK conflict" is if the values for ExportID were all different (like in `fix_incorrect_personal_id_references` below). Since the value is all the same here, a simple update statement seems much better to me. It's less code/complexity and will be more performant

I'd keep the logging though, that seems good

Reply:
Oh, yeah, sorry, I mis-read that and thought we were doing more than just setting all the ExportID to nil.  I agree, we don't even need to batch this, just 
```
Hmis::Hud::Enrollment.hmis.where.not(ExportID: nil).update_all(ExportID: nil)
```

Reply:
it's updated